### PR TITLE
Hello Nicolas

### PR DESCRIPTION
**Pull request by MikePlace1971**
Ich hoffe das passt jetzt so ;-). Ich habe erstmal nur die readme.md geändert.

### CODE REVIEW

**Comment by seppemarotta** [comment]
Interesting, I might check your branch with your improvements.

**Comment by seppemarotta** [comment]
I've been trying to match this with the current PS setup but I wasn't able to do so. Have you been running this on PP or PS ?

**Comment by dickreuter** [comment]
Ok great. Can you create a pull request?

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Deep mind pokerbot for pokerstars and partypoker
 
+Hi,
+I will try to improve the reading of the values. It will no longer be necessary to define coordinates.txt exactly. It is enough a generous range around the value. For this, I have extracted the numbers individually for cv2 as .png. The whole thing I have adapted only for PartyPoker.
```
[the remark this answers is not in the document]
Comment (dickreuter) [reply readme.md]: Great job, can you please clean up the readme.md to reflect the current status. thanks

**Pull request merged**